fr/lecons/normaliser-donnees-textuelles-python.md

```diff
@@ -105,6 +113,22 @@ wordlist = text.split()
 print(wordlist)
 ```
 
+<div class="alert alert-warning">
+Si vous préférez suivre cette leçon en continuant d'utiliser notre exemple du Old Bailey Online, il vous faudra d'abord <a href="/assets/normaliser-donnees-textuelles-python/obo-t17800628-33.html">télécharger le fichier HTML de la page en question</a>, que nous vous avons mis à disposition, puis adapter les premières lignes du code pour pointer vers le fichier sur votre propre ordinateur, comme ceci&nbsp;:
```

Comment:
I think is clear. Yes, the html file won’t download automatically, it will open in a new tab. Direct download only happens with `.zip` or other files which the browser cannot render. As I understand it, where we link to a single file with a raw format that the browser can display, it will do so. From here, readers can download that html file as you explain _télécharger le fichier HTML de la page en question_.

Reply:
This is a clear statement to me. I was just thinking of 2 other solutions: 
- use `urlopen` on the local file. This way, we don't have to suggest a workaround with `open()` and `read()`
- why not use the PH url of the old baily archive ? Something like  `https://deploy-preview-3350--ph-preview.netlify.app/assets/normaliser-donnees-textuelles-python/obo-t17800628-33` (but not the preview)? Would it work ?

Reply:
To your first question: Could you test this, @matgille?

To your second question: Yes! The link Charlotte is suggesting here is to the HTML file where it would be available in our repository.

Reply:
Ok, so the first option seems to work on my computer: 
```python
#html-to-list1.py
import urllib.request, urllib.error, urllib.parse

url = 'file:///home/mgl/Téléchargements/PH.html'

response = urllib.request.urlopen(url)
html = str(response.read().decode('UTF-8'))
print(html)
```

This snippet outputs the content of the HTML file I've downloaded from the link you provide in your edits, @charlottejmc (thanks!)

Reply:
Maybe the second option would be even simpler ?

Reply:
As we've discussed, this option of replacing a live URL in the code with a link to a pre-prepared html file means that a fundamental learning action of the lesson (importing html from a live website) is being missed. I think that needs to be clear to our readers.

Reply:
@matgille, 

1- It's interesting to hear that `urlopen` would work on the downloaded file. However, I worry that this might be introducing too many concepts at once in what is supposed to be a beginner lesson. 
2- If this works, I like the idea of simply using the URL to the archived HTML file saved in our assets (currently only the `deploy` link would work because we haven't merged the branch, but later the usual `github/programminghistorian` would replace it). This does seem to be the method which comes the closest to the original lesson objective.